tests/test_gui.py

```diff
@@ -44,15 +44,18 @@ def context():
     )
 
     # Make sure there is not already an automation project created
-    filters = [["name", "is", "Toolkit Loader2 UI Automation"]]
+    project_name = (
+        "Toolkit Loader2 UI Automation " + os.environ["SHOTGUN_TEST_ENTITY_SUFFIX"]
```

Comment:
This pattern repeats itself in all your repos. Maybe we should refactor it.

in tk-toolchain, could you add a new file called `tk_toolchain/testing.py`. In it, you add a new method, some like 
create_unique_name. Takes a single parameter, the name and then adds the content of the env var if set.

You can look at other files to see how we document methods and tests to see how we turn on/off env vars during a single test.